blivet/devicelibs/stratis.py

```
@@ -246,6 +248,14 @@ def create_pool(name, devices, encrypted, passphrase, key_file, clevis, overprov
     except DBusError as e:
         raise StratisError("Failed to enable overprovisioning on stratis pool: %s" % str(e))
 
+    if fs_limit and fs_limit != STRATIS_FS_LIMIT_DEFAULT:
+        try:
+            proxy = util.SystemBus.get_proxy(STRATIS_SERVICE, pool_path,
+                                             STRATIS_POOL_INTF)
+            proxy.FsLimit = fs_limit  # pylint: disable=assigning-non-slot
+        except DBusError as e:
+            raise StratisError("Failed to set filesystem limit on stratis pool: %s" % str(e))
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)